frontend/src/pages/EventSignInPage/styles.ts

```diff
@@ -12,6 +12,16 @@ const useStyles = makeStyles({
     alignItems: 'center',
     width: '600px',
     paddingBottom: '30px',
+    [theme.breakpoints.up('sm')]: {
+      width: 600,
+      marginLeft: 'auto',
+      marginRight: 'auto',
+    },
+    [theme.breakpoints.down('sm')]: {
```

Comment:
Just out of curiosity, what does the up() and down() thing do? And what is the `breakpoints` thing? I've always been bad with CSS so just wanted to know : )

Reply:
it's creating different size media query based on the screen size, up would be targeting screen size larger than the given parameter, and down would be for screen size smaller than the parameter